### DISCUSSION TITLE
更新到231003不能用

### DISCUSSION THREAD

**Discussion by lwh201314** [MonwF/customiuizer#375] [Q&A]
红米k50  miui14  lsposed1.8.6

230826是正常的，更新以后所有修改都失效了，清除了数据恢复备份也没用，重启的功能也不能用。

**Comment by MonwF** [comment]
有没激活先，再恢复备份？

**Reply by lwh201314** [reply]
都试过不行，就差没把lsp更新了

**Reply by MonwF** [reply]
。。。你在哪里下载的，不看更新日志吗？明确说明1003版本必须lsp在1.9.1以上

**Reply by lwh201314** [reply]
😅知道了，确实要1.9.1，没注意看日志
